src/cf-layout/src/cf-layout.less

```diff
@@ -346,16 +346,35 @@
 
     &__flush-top {
         margin-top: 0 !important;
+
+        &.block__border,
+        &.block__border-top {
+            border-top: none;
+        }
     }
 
     &__flush-bottom {
         margin-bottom: 0 !important;
+
+        &.block__border,
+        &.block__border-bottom {
+            border-bottom: none;
+        }
     }
 
     &__flush-sides {
         margin-right: -(@grid_gutter-width / 2);
         margin-left: -(@grid_gutter-width / 2);
 
+        .respond-to-max(@bp-xs-max, {
```

Comment:
Because the page has max-width, so it's not truly "flush". We still want borders on the sides. (If there's a sidebar, the way we're drawing the sidebar border places it overtop of the block's border.)

Check out a couple live examples on this page: http://www.consumerfinance.gov/adult-financial-education/

As to the think you linked: Maybe? Typically `border-width: 0` and `border: none` have the same effect.